partials/position.html

```diff
@@ -1,16 +1,16 @@
 <div id="positionField" class="o-forms o-forms--wide ncf__field js-field{{#if hasError}} o-forms--error{{/if}}" data-ui-item="select" data-ui-item-name="position" data-validate="required">
 
-	<label for="position" class="o-forms__label">What's your job position?</label>
+	<label for="position" class="o-forms__label">What&#x27;s your job position?</label>
```

Comment:
Again some weird quote, intended?
```suggestion
	<label for="position" class="o-forms__label">What's your job position?</label>
```

Reply:
Can't do this cause React convert the `'` into an entity (&#x27;), that is appropriate, and this will break the test.